README.md

```diff
@@ -1,84 +1,147 @@
 
+### 1. Predict and log
 
-This is an example for logging model predictions from a 🤗 transformers text classification pipeline:
+Let's load the zero-shot pipeline and the dataset (we are using the AGNews dataset for demonstration, but this could be your own dataset). Then, let's go over the dataset records and log them using `rb.log()`. This will create a Rubrix dataset, accesible from the UI.
 
 ```python
 from transformers import pipeline
 from datasets import load_dataset
 import rubrix as rb
 
-model = pipeline('zero-shot-classification', model="typeform/squeezebert-mnli")
+model = pipeline('zero-shot-classification', model="typeform/distilbert-base-uncased-mnli")
 
 dataset = load_dataset("ag_news", split='test[0:100]')
 
-# Our labels are: ['World', 'Sports', 'Business', 'Sci/Tech']
-labels = dataset.features["label"].names
+labels = ['World', 'Sports', 'Business', 'Sci/Tech']
 
 for record in dataset:
     prediction = model(record['text'], labels)
 
     item = rb.TextClassificationRecord(
         inputs=record["text"],
-        prediction=list(zip(prediction['labels'], prediction['scores'])),
-        annotation=labels[record["label"]]
+        prediction=list(zip(prediction['labels'], prediction['scores']))
     )
 
-    rb.log(item, name="ag_news_zeroshot")
+    rb.log(item, name="news_zeroshot")
 ```
 
-# Quick links
+### 2. Explore, Filter and Label
+
+Now we can access our Rubrix dataset and start annotating data. Let's filter the records predicted as `Business` with high probability and use the bulk-labeling feature for labeling 15 records as `Business`:
+
+https://user-images.githubusercontent.com/1107111/132261244-b9151571-608e-4a41-8f34-e9dc1c8b8e38.mp4
+
+
+### 3. Load and create a training set
+After a round of data annotation, we can load the Rubrix dataset and transform it into a 🤗 dataset for training a supervised classifier.
+
+```python
+from datasets import Dataset
+
+rb_df = rb.load(name='news_zeroshot')
+
+# filter annotated records
+rb_df = rb_df[rb_df.status == "Validated"]
+
+# select text input and the annotated label
+rb_df['text'] = rb_df.inputs.transform(lambda r: r['text'])
+
+# labels can be a list (for supporting multi-label text classifiers)
+rb_df['labels'] = rb_df.annotation.transform(lambda r: r[0])
+
+
+# create 🤗 dataset from pandas with labels as numeric ids
+label2id = {label: i for i,label in enumerate(labels)}
+train_ds = Dataset.from_pandas(rb_df[['text', 'labels']])
+train_ds = train_ds.map(lambda example: {'labels': label2id[example['labels']]})
+```
+
+## Architecture
+
+Rubrix main components are:
+
+- **Rubrix Python library**: Python client to log, load, copy and delete Rubrix datasets from Python.
+- **Rubrix server**: FastAPI REST service for reading and writing data.
+- **Elasticsearch**: The storage layer and search engine powering the API and UI.
+- **Rubrix UI**: Easy-to-use UI for data exploration and annotation.
```

Comment:
Should we stick with the term `Rubrix UI`? In the docs we often refer to this component as `Web App`. Either way, I think we should be consistent.

Reply:
You're right, can you change it to Web App here?